src/Propel/Runtime/ServiceContainer/StandardServiceContainer.php

```diff
@@ -107,6 +108,11 @@ class StandardServiceContainer implements ServiceContainerInterface
      */
     protected $loggerConfigurations = [];
 
+    /**
+     * @var bool | null
```

Comment:
Surprised the cs doesnt complain here

Reply:
Surprised me too, thought it had to be `mixed`

Reply:
I refer to the whitespace, should be bool|null